tests/karate/src/test/java/fe/utils/web.feature

```diff
@@ -44,6 +55,19 @@ Feature: web test
             }
           """
 
-    And click(tab_launch_selector)
-    And input(tab_launch_address_selector, backendURL)
-    And click(tab_home_selector)
+    * click(exploring_selector)
+    * click(tab_connect_selector)
+    * click(launch_selector)
+    * input(launch_address_selector, backendURL)
```

Comment:
not sure about the variable naming, it's not immediately obvious what this actually does.